pkg/database/handler.go

```diff
@@ -221,6 +225,99 @@ func (h Handler) SaveAs(c *gin.Context) {
 	c.JSON(http.StatusCreated, save)
 }
 
+// Save database
+func (h Handler) Save(c *gin.Context) {
+	// swagger:route POST /databases/save/{instanceId} saveDatabase
+	//
+	// Save database
+	//
+	// Saving a database involves the following steps
```

Comment:
I don't think we need this doc 😄  This is clear from the `Save` method in the service. If you want to describe this than it should be on the `Save` method in the service

Reply:
The benefit of putting it on the handler is that it'll be included in our swagger generated api documentation. But maybe we don't need it after all. One downside is that we have to keep the comment in sync with the code.

@radnov, @Philip-Larsen-Donnelly - What do you guys think?

Reply:
These are implementation details to me. Users of the API should care about the end result not how we achieve it. This is why I would not document the steps. You also duplicate the code you wrote and thus get into the trouble of it getting out of sync.